ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -472,15 +472,16 @@ private NamedBlobRecord run_get(String accountName, String containerName, String
         String blobId = Base64.encodeBase64URLSafeString(resultSet.getBytes(1));
         Timestamp expirationTime = resultSet.getTimestamp(2);
         Timestamp deletionTime = resultSet.getTimestamp(3);
+        if (expirationTime != null && expirationTime.before(deletionTime)) {
```

Comment:
I can't use timestampToMs and then do min(ts1, ts2), because the timestampToMs will map null to -1. While in this case, we want to treat null as Inf. I just created a `getMinTimestamp` method to handle that properly